tasks/test-project/test-project

```diff
@@ -348,28 +347,23 @@ const globalTasks = () =>
         title: 'Lint --fix all the things',
         task: async () => {
           try {
-            await execa(
-              'yarn rw lint --fix',
-              [],
-              // getExecaOptions(OUTPUT_PROJECT_PATH)
-              {
-                shell: true,
-                stdio: 'ignore',
-                cleanup: true,
-                cwd: OUTPUT_PROJECT_PATH,
-                env: {
-                  RW_PATH: path.join(__dirname, '../../'),
-                },
-              }
-            )
+            await execa('yarn rw lint --fix', [], {
+              shell: true,
+              stdio: 'ignore',
+              cleanup: true,
+              cwd: OUTPUT_PROJECT_PATH,
+              env: {
+                RW_PATH: path.join(__dirname, '../../'),
+              },
+            })
           } catch (e) {
             // nothing to see here
           }
         },
         enabled: () => rebuildFixture,
```

Comment:
```suggestion
        enabled: () => rebuildFixture || !copyFromFixture,
```

@dac09 I ran `yarn build:test-project --no-copyFromFixture` and ran into a lint error from jscodeshift codemods. After I ran `yarn rw lint --fix` on the generate project, it worked fine.

To confirm --> this updated enabled logic should fix that, correct?